uktrade/data-hub-frontend
# Improve audit view for Investment Projects

## Description of change
Uses the existing Edit-History component to display the investment project audit data returned by the api. This is needed because the current audit view for investment projects only showed limited data. 

Also moves the EditHistory component and required data transformers to generic directories.

## Test instructions
* Visit Investments, click on a project, then select 'Audit Summary' from the left-hand bar
* You should see an updated audit history (as in the screenshot below).
* If there are no changes, make changes to the investment project then check the audit history again.

## Screenshots
### Before
<img width="986" alt="Screenshot_2020-04-21_at_11 50 42" src="https://user-images.githubusercontent.com/23265724/80352487-dda9ea80-886b-11ea-8137-80bf22ed1597.png">

### After
<img width="919" alt="Screenshot 2020-04-27 at 09 53 10" src="https://user-images.githubusercontent.com/23265724/80353214-f1a21c00-886c-11ea-90fe-a1327c8e492d.png">

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
